src/lib/fauna-account-client.mjs

```diff
@@ -198,20 +200,20 @@ export class FaunaAccountClient {
    * @param {Object} params - The parameters for creating the key.
    * @param {string} params.path - The path of the database, including region group
    * @param {string} params.role - The builtin role for the key.
-   * @param {string} params.ttl - ISO String for the key's expiration time
+   * @param {string | undefined} params.ttl - ISO String for the key's expiration time, optional
+   * @param {string | undefined} params.name - The name for the key, optional
    * @returns {Promise<Object>} - A promise that resolves when the key is created.
    * @throws {Error} - Throws an error if there is an issue during key creation.
    */
-  async createKey({ path, role, ttl }) {
-    const TTL_DEFAULT_MS = 1000 * 60 * 60 * 24;
-    return await this.retryableAccountRequest({
+  async createKey({ path, role, ttl, name }) {
+    return this.retryableAccountRequest({
       method: "POST",
       path: "/databases/keys",
       body: JSON.stringify({
         role,
         path: FaunaAccountClient.standardizeRegion(path),
-        ttl: ttl || new Date(Date.now() + TTL_DEFAULT_MS).toISOString(),
-        name: "System generated shell key",
+        ttl,
```

Comment:
do we want to use the default TTL computed above (and commented out)? if so, let's; if not, let's trash the comment

Reply:
I think i deleted that already?

Reply:
I don't see it anywyhere in this file: https://github.com/fauna/fauna-shell/blob/d84e9db87d9d014b3a8f21d77938b63525807698/src/lib/fauna-account-client.mjs